### DISCUSSION TITLE
모던 자바스크립트 19-7장~19.10장

### DISCUSSION THREAD

**Discussion by pipisebastian** [BDD-CLUB/challenge-every-week-posting#271] [1. 개발 관련 글 모음]
```
⁽⁽◝(˙꒳˙)◜⁾⁾ 날아가욥~

________________⁽⁽◝(˙꒳˙)◜⁾⁾_____도착했어요
```
[블로그링크입니다!](https://velog.io/@pipi/%EB%AA%A8%EB%8D%98-%EC%9E%90%EB%B0%94%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8-Deep-Dive-19.719.9%EC%9E%A5)

**Comment by mun-kyeong** [comment]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 이모티콘 넘나 귀여운것,,

**Reply by pipisebastian** [reply]
훗.. 열심히 찾았습니다^^..
